File: server/src/main/java/cn/iinti/atom/entity/metric/MetricHour.java
package cn.iinti.atom.entity.metric;

import com.baomidou.mybatisplus.annotation.TableName;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

/**
 * <p>
 * 监控指标,小时级
 * </p>
 *
 * @author iinti
 * @since 2023-03-13
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
@TableName("metric_hour")
@Schema(name =  "Metric对象", description = "监控指标")
public class MetricHour extends Metric {


}

File: server/src/main/kotlin/cn/iinti/katom/entity/metric/MetricHour.kt
package cn.iinti.katom.entity.metric

import com.baomidou.mybatisplus.annotation.TableName
import io.swagger.v3.oas.annotations.media.Schema
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.experimental.Accessors

@Schema(name = "Metric对象", description = "监控指标,小时级")
@TableName("metric_hour")
@Accessors(chain = true)
@Data
@EqualsAndHashCode(callSuper = false)
class MetricHour : Metric() {

}